packages/cli/src/bin.ts

```diff
@@ -182,6 +182,15 @@ async function main(): Promise<number> {
     return 1
   }
 
+  const checkResult = await runCheckpointClientCheck({
```

Comment:
Can we just kick this off here, but await only later? So we won't block user interaction for network requests or so?